### PR TITLE
Completed array methods exercise. - Tiffany Brown

### PR DESCRIPTION
**Pull request by tiffanydbrown**
Screenshot 2023-12-04 at 3 43 39 PM

### CODE REVIEW

File: array-methods/for-each.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+console.log('In order:');
+values.forEach((element) => console.log(element));
+
+const reversedArray = [];
+
+values.forEach((element) => {
+  reversedArray.unshift(element);
+});
+console.log('Reverse order:');
+reversedArray.forEach((element) => console.log(element));
```
Comment (shawnkost) [inline array-methods/for-each.js:12]: You should be accomplishing this without the use of a 2nd array

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Tiffany

**Comment by tiffanydbrown** [comment]
Screenshot 2023-12-04 at 6 51 38 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**